YDS-Storybook/SwiftUI/Atom/BadgePageView.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  BadgePageView.swift
+//  YDS-Storybook
+//
+//  Created by 성현주 on 10/31/23.
+//
+
+import SwiftUI
+import YDS_SwiftUI
+
+
+
+struct BadgePageView: View {
+    let title: String = "BadgeView"
+    
+    @State var text: String? = "광고"
+    @State var badgeColorSelectedIndex = 0
+    @State var icons: SwiftUIIcon? = YDSSwiftUIIcon.icons[0]
+    
+    var selectedColor: Color {
+        return YDSItemColor.allCases[badgeColorSelectedIndex].backgroundColor
+    }
+
+    
+    public var body: some View {
+        StorybookPageView(sample: {
+            VStack {
+                GeometryReader { geometry in
+                    YDSBadge(
+                        text: text ?? nil,
```

Comment:
```suggestion
                        text: text,
```
StorybookPageView에서는 옵션을 끄면 자동으로 nil 값이 들어가게 되서 옵셔널 값을 굳이 찾지 않아도 동일하게 작동해요

Reply:
넵 수정하겠습니다